applications/stable-asset.md

```diff
@@ -0,0 +1,142 @@
+# W3F Open Grant Proposal
+
+* **Project Name:** Stable Asset
+* **Team Name:** NUTS Finance
+* **Payment Address:** 0x679824d755B054a2a50358008472a6F400740319
+
+## Project Overview :page_facing_up:
+
+### Overview
+
+There are vastly emerging assets in the Polkadot ecosystem, including both Polkadot native assets and assets bridged from other blockchains such as Ethereum. These assets introduce diversity in architecture and business model, but also fragmentizes the ecosystem since applications need to build separate markets for each of these assets.
+
+Stable Asset is a synthetic protocol for assets of specific peg such as BTC and USD. By creating synthetic asset which is backed by a basket of assets with the same value peg, ACoconut unifies market values of multiple assets and simplifies financial application development since only one synthetic asset needs to be supported for each peg type.  Stable Asset is also built with integrated swap and saving functionalities using the basket of assets.
```

Comment:
Isn't it highly unlikely that any chain drops support for all widespread synthetic assets in favour of yours, or what do you mean by this?

Reply:
I think you beat me to it ;)

Reply:
The Stable Asset protocol is a general asset synthesis protocol so that there is no asset that is in favor of it. In short, it should work with any assets on any chain.

Reply:
This was just amended, see [`14c320b#L13`](https://github.com/w3f/Open-Grants-Program/pull/286/commits/14c320bcac8c98217e8383cb50d1d8cd98b65fae#diff-118b14b0a1f561ce87e7bb1f372478aa07697b02733620b8957685562b038673R13)